fre/make/tests/null_example/test_checkout_null_yaml.py

```diff
@@ -0,0 +1,45 @@
+#tests for the create-checkout step of fre-make, for null_model.yaml
+import os
+from fre import fre
+from pathlib import Path
+from fre.pp import configure_script_yaml as csy
+from click.testing import CliRunner
```

Comment:
Can remove these too, not needed. We won't be using `runner.invoke` for these tool tests here